great_expectations/validation_operators/validation_operators.py

```diff
@@ -22,11 +26,47 @@ class ValidationOperator(object):
     """
     The base class of all validation operators.
 
-    It defines the signature of the public run method - this is the only
-    contract re operators' API. Everything else is up to the implementors
+    It defines the signature of the public run method. This method and the validation_operator_config property are the
+    only contract re operators' API. Everything else is up to the implementors
     of validation operator classes that will be the descendants of this base class.
     """
 
+    def __init__(self) -> None:
+        self._validation_operator_config = None
+
+    @property
+    def validation_operator_config(self):
+        """
+        This method builds the config dict of a particular validation operator. The "kwargs" key is what really
+        distinguishes different validation operators.
+
+        e.g.:
+        {
+            "class_name": "ActionListValidationOperator",
+            "module_name": "great_expectations.validation_operators",
+            "name": self.name,
+            "kwargs": {
+                "action_list": self.action_list
+            },
+        }
+
+        {
```

Comment:
Is this the right approach for validation operator (or checkpoint) config?

Reply:
I'm not sure - personally, this would be easier to discuss synchronously.